src/test/java/com/openlattice/shuttle/transforms/TransformTest.java

```diff
@@ -24,6 +24,8 @@
         testrow.put("SSN", null);
         testrow.put("Address", "560 Scott Street, San Francisco, CA 94117");
         testrow.put("CommittedDateTime", "03/05/00 10:00");
+        testrow.put("Latt", "36.23452");
```

Comment:
this is the most trivial comment ever and this isn't even real data, but you might change this to "Lat" or "Latit" since latitude isn't spelled with a double t